### PR TITLE
OGM-1205 Occasional failures of SequenceIdGeneratorTest for the Hot R…

### PR DESCRIPTION
**Pull request by Sanne**
…od dialect

https://hibernate.atlassian.net/browse/OGM-1205

### CODE REVIEW

File: infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/sequences/SequencedActor.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ */
+@Entity
+public class SequencedActor {
+	static final transient int INITIAL_VALUE = 1;
+	private Long id;
+	private String name;
+	private String totalMovies;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequencerActorGenerator")
```
Comment (gsmet) [inline infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/test/sequences/SequencedActor.java]: s/sequencer/sequenced/

**Comment by gsmet** [comment]
I'll fix the typos and merge once CI is green.

**Comment by Sanne** [comment]
I really meant "sequencer" but ok it's the same ;)

**Comment by gsmet** [comment]
@Sanne it just feels a bit weird to have the entity named `Sequence**d**Actor` and the generator `sequence**r**ActorGenerator`.

I think we should be consistent. If you agree, pick your choice, I'll do the leg work. If not, just push it :).

**Comment by Sanne** [comment]
ok, applied your suggestions and pushed it.

**Pull request closed without merging**